docs/technical-reference/api/moduleConfig-cr.md

```diff
@@ -0,0 +1,97 @@
+# ModuleConfig Custom Resource
+
+The [ModuleConfig custom resource (CR)](../../../api/v1alpha1/moduleconfig_types.go) facilitates the synchronization of `Module` CR configuration between the SKR and KCP clusters, reflecting dynamic changes made by the user.
+
+## Generation
+
+The SKR Watcher notifies the Lifecycle Manager (LM) in case of changes detected in a `Module`. A `ModuleConfig` is then generated/updated by the LM in from the `Module` within the SKR cluster.
```

Comment:
```suggestion
[Runtime Watcher](https://github.com/kyma-project/runtime-watcher) notifies Lifecycle Manager (LM) of changes detected in a Module CR. Lifecycle Manager generates or updates a ModuleConfig CR in Kyma runtime, based on the detected changes.
```
- Do you mean Runtime Watcher?
- Are the changes detected by Runtime Watcher visible in the ModuleConfig CR configuration?